bfdd/bfdd_cli.c

```diff
@@ -657,6 +1012,182 @@ DEFPY_YANG(bfd_peer_profile, bfd_peer_profile_cmd,
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY(
+	sbfd_reflector, sbfd_reflector_cmd,
+	"sbfd reflector source-address X:X::X:X$srcip discriminator WORD...",
+    "seamless BFD\n"
+    "sbfd reflector\n"
+	"binding source ip address\n"
+	IPV6_STR
+	"discriminator\n"
+	"discriminator value or range (e.g. 100 or 100 200 300 or 100-300)\n")
+{
+	int idx_discr = 5;
+	int i;
```

Comment:
with DEFPY, you dont need to account for the argument number, you can use WORD$disc , to get it.